integration/PetPortalServiceNowExtended/xebialabs/secrets.xlvals.tmpl

```diff
@@ -0,0 +1,4 @@
+# This file includes all secret values, and will be excluded from GIT. You can add new values and/or edit them and then refer to them using '!value' YAML tag
+XL_DEPLOY_PASSWORD=admin
```

Comment:
As I mentioned earlier this file is autogenerated I added below to DoD

Do not add the `xebialabs/.gitignore`, `xebialabs/values.xlvals` and `xebialabs/secrets.xlvals` to the `bleuprint.yaml` files section, and do not check in those files, as they are autogenerated based on parameter definitions. Secrets are created when a parameter in the `blueprint.yaml` has the `secret: true` directive. For adding parameters to `values.xlvals` use the `saveInXlVals: true` directive.

If you want to add something to this file but dont want to ask a question then add it to the parameters section and just give it a value like(Please note it will work only with new CLI version)

```
- name: XL_DEPLOY_PASSWORD
  type: Input
  secret: true
  value: admin
```